### DISCUSSION TITLE
ERROR: 'USER'

### DISCUSSION THREAD

**Discussion by yuanjie-ai** [microsoft/nni#3795] [Q&A] [answered]
# /usr/local/bin/nnictl create --config /fds/2_Competitions/A/nni/examples/trials/mnist-annotation/config_remote.yml
INFO:  expand codeDir: . to /fds/2_Competitions/A/nni/examples/trials/mnist-annotation/. 
INFO:  Starting restful server...
ERROR: 'USER'

**Comment by JunweiSUN** [comment]
Hello @Jie-Yuan Seems like your config_remote.yml has something wrong. Can you post your config_remote.yml？

**Comment by scarlett2018** [comment]
> Hello @Jie-Yuan Seems like your config_remote.yml has something wrong. Can you post your config_remote.yml？

@Jie-Yuan  - may you post the config_remote.yml as @JunweiSUN suggested?

**Comment by YOOyong** [comment]
got same error on google colab.
other nni examples work. 
only mnist-annotation example shows  ERROR: 'USER'

**Comment by robscc** [comment, marked as answer]
got same error on local docker,
coz annotation mode would get environment 'USER', so it raise error.
to fix this is easy, 
`export USER=root`

[https://github.com/microsoft/nni/blob/9d0b6fa60e798028e74fce227f8d83e993046256/nni/tools/nnictl/common_utils.py#L79](url)
